Kryptonite/Util.swift

```diff
@@ -82,7 +82,20 @@ extension Data {
             throw UTF8EncodingError()
         }
         return utf8String
-    }    
+    }
+    
+    func stripLeadingZeros() -> Data {
+        guard self.count > 0 else {
```

Comment:
I think this can be simplified to `return self.drop(while: {$0 == 0})`

Reply:
don't need the function anymore, so will remove altogether